src/components/calcite-pick-list/shared-list-logic.ts

```diff
@@ -151,7 +151,7 @@ export function keyDownHandler<T extends Lists>(this: List<T>, event: KeyboardEv
 }
 
 export function internalCalciteListChangeEvent<T extends Lists>(this: List<T>): void {
-  this.calciteListChange.emit(this.selectedValues);
+  this.calciteListChange.emit(this.selectedValues as any);
```

Comment:
Can this be typed more or no? Just wondering

Reply:
I'll do that in the follow-up issue I mentioned. I couldn't get TS to stop complaining by using the conditional types.